docker/README.md

```diff
@@ -34,6 +34,7 @@ Use `build_image.sh` script to build the docker images. The script builds the `p
 |-h, --help|Show script help|
 |-b, --branch_name|Specify a branch name to use. Default: master |
 |-g, --gpu|Build image with GPU based ubuntu base image|
+|-bi, --baseimage specify base docker image. Example: nvidia/cuda:11.7.0-cudnn8-runtime-ubuntu20.04|
```

Comment:
What is the reason for using CUDA 11.7? Can we switch to 11.8?

Reply:
Currently, all our infra/tests are on CUDA 11.7. This was targeted for 0.9.0 . Created a ticket to track this. https://github.com/pytorch/serve/issues/2486

Reply:
Updated to CUDA 11.8